tools/yarn.lock

```diff
@@ -136,40 +136,37 @@
     universal-github-app-jwt "^2.2.0"
     universal-user-agent "^7.0.0"
 
-"@octokit/auth-oauth-app@^7.1.0":
```

Comment:
Do you know why these changes of the `yarn.lock` are present here in your PR? Usually, this file should only be modified if dependencies in a package.json are added/removed/upgraded.
Please revert this file if you did not actively or purposefully changed this file.

Reply:
Ah no. I didn't change it on purpose. I will revert the changes. 
Thanks for the notice :)